app/controllers/user_challenges_controller.rb

```diff
@@ -27,6 +28,28 @@ def destroy
     end
   end
 
+  def evaluate_comments
+    evaluator = UserChallenge::Evaluator.new(params: params)
+    result = evaluator.save_comments
+
+    if result.success?
+      head :no_content
+    else
+      render json: { resource: 'user_challenge', errors: [user_challenge: result.errors.first] }, status: 500
```

Comment:
On GitHub we raise a [`422`](https://httpstatuses.com/422) for something like this